predicators/explorers/active_sampler_explorer.py

```diff
@@ -154,18 +165,19 @@ def _update_ground_op_hist(self, state: State) -> None:
         if nsrt is None:
             return
         atoms = utils.abstract(state, self._predicates)
-        success = nsrt.add_effects.issubset(atoms) and \
-            not nsrt.delete_effects & atoms
-        logging.debug(f"[Explorer] Last NSRT: {nsrt.name}{nsrt.objects}")
-        logging.debug(f"[Explorer]   outcome: {success}")
+        success = nsrt.add_effects.issubset(atoms)
```

Comment:
This isn't quite right as a check either! We should really be using the necessary atoms here. Probably want to update with a comment.